src/Catty/ViewController/Continue&New/MaintainObject/MaintainLooks/LooksTableViewController.m

```diff
@@ -725,41 +726,6 @@ - (void)setupEditingToolBar
 }
 
 #pragma mark paintDelegate
-
-- (void)showSavePaintImageAlert:(UIImage *)image andPath:(NSString *)path
```

Comment:
`Class 'LooksTableViewController' does not conform to protocol 'PaintDelegate'`. Please change the delegate and remove unused methods.